lib/web3/contract.js

```diff
@@ -117,8 +117,9 @@ var checkForContractAddress = function(contract, callback){
             } else {
 
                 contract._eth.getTransactionReceipt(contract.transactionHash, function(e, receipt){
-                    if(receipt && !callbackFired) {
+                    if(receipt && receipt.blockHash && !callbackFired) {        //  sometimes blockHash is null the second time - Marco Guarducci <mark.guarducci@gmail.com>
```

Comment:
There's no particular reason you would add this comment. It's already in git and it doesn't clarify anything particularly.

Reply:
yes please remove that comment and we can merge that in. web3.js should not be a self advertisement platform ;P

Reply:
i have removed comment